### PR TITLE
fix: improve code highlight style, close #222, #223

### LINKED ISSUE #222: [Bug report] Code group has unreadable background in light mode
### Description

Try to use https://ecosystem.vuejs.press/themes/default/components.html#codegroupitem after switching to the Light mode.

code-group

### Reproduction

https://ecosystem.vuejs.press/themes/default/components.html#codegroupitem

### Used Package Manager

npm

### System Info

```shell
123
```

### LINKED ISSUE #223: Plugin cannot control highlight lines
### Description

```
        themePlugins: {
            mediumZoom: false,
            prismjs: {
                theme: 'one-light',
                highlightLines: true,
                lineNumbers: true
            }
        },
```

Not working

```
        shikiPlugin({
            // 配置项
            langs: ['ts', 'json', 'vue', 'md', 'bash', 'diff', 'java'],
            theme: 'github-light',
            lineNumbers: true,
            highlightLines: true
        }),
```

Not working too

### Reproduction

https://stackblitz.com/edit/vuepress-lgyjeu?file=docs%2F.vuepress%2Fconfig.ts

### Used Package Manager

npm

### System Info

```shell
{
  "name": "",
  "version": "0.0.1",
  "description": "",
  "license": "MIT",
  "type": "module",
  "scripts": {
    "dev": "vuepress dev docs",
    "build": "vuepress build docs",
    "docs:clean-dev": "vuepress dev docs --clean-cache",
    "update-package": "npx vp-update"
  },
  "devDependencies": {
    "@vuepress/bundler-vite": "2.0.0-rc.14",
    "@vuepress/plugin-markdown-math": "2.0.0-rc.40",
    "@vuepress/plugin-search": "2.0.0-rc.40",
    "@vuepress/theme-default": "2.0.0-rc.40",
    "mathjax-full": "3.2.2",
    "vue": "3.4.34",
    "vuepress": "2.0.0-rc.14"
  }
}
```

### PR DESCRIPTION
**Pull request by pengzhanbo**
### Before submitting the PR, please make sure you do the following 

- [ ] Read the [Contributing Guidelines](https://github.com/vuepress/ecosystem/blob/main/CONTRIBUTING.md).
- [ ] Provide a description in this PR that addresses **what** the PR is solving. If this PR is going to solve an existing issue, please reference the issue (e.g. `close #123`).

### What is the purpose of this pull request? 

- [x] Bug fix
- [ ] New feature
- [ ] Other

### Description

close #222 
close #223

### CODE REVIEW

File: themes/theme-default/src/client/components/global/CodeGroup.ts [hunk 1]
```diff
@@ -30,6 +31,35 @@ export const CodeGroup = defineComponent({
...
+            )
+            groupRef.value!.style.setProperty(
+              '--c-code-group-tab-title',
+              val ? darkColor : lightColor,
+            )
+          },
+          { immediate: true },
+        )
+      }
+    })
```
Comment (Mister-Hope) [inline themes/theme-default/src/client/components/global/CodeGroup.ts:62]: Does the code work with prismjs? I did not see any fallback from it.

File: themes/theme-default/src/client/components/global/CodeGroup.ts [hunk 2]
```diff
@@ -127,35 +157,42 @@ export const CodeGroup = defineComponent({
...
-                  }
-                },
-                class: {
-                  'code-group-nav-tab': true,
-                  'active': isActive,
+      return h(
+        'div',
+        {
+          class: 'code-group',
+          ref: (el) => (groupRef.value = el as HTMLDivElement),
```
Comment (Mister-Hope) [inline themes/theme-default/src/client/components/global/CodeGroup.ts:164]: How does a direct assign `el: groupRef` not working?

File: themes/theme-default/src/client/components/global/CodeGroup.ts (hunk 1 above)
Comment (pengzhanbo) [reply themes/theme-default/src/client/components/global/CodeGroup.ts:62]: PrismJS is functioning properly and does not require JavaScript to calculate styles.

File: themes/theme-default/src/client/components/global/CodeGroup.ts (hunk 2 above)
Comment (pengzhanbo) [reply themes/theme-default/src/client/components/global/CodeGroup.ts:164]: I forgot that I could write like this. 😂

**Pull request merged**